frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/os-flavor.tsx

```diff
@@ -69,6 +90,7 @@ export const OSFlavor: React.FC<OSFlavorProps> = React.memo(
       ? {
           userTemplates,
           commonTemplates,
+          commonDataVolumes,
```

Comment:
the same should be also done for wokload as it can trigger the not ready updater (needs commonDataVolumes)

Reply:
added

Reply:
removed, it blocks the page from rendering

Reply:
done